providers/common/io/tests/unit/common/io/xcom/test_backend.py

```diff
@@ -24,14 +24,20 @@
 import airflow.models.xcom
 from airflow.providers.common.io.xcom.backend import XComObjectStorageBackend
 from airflow.providers.standard.operators.empty import EmptyOperator
-from airflow.utils import timezone
+
+try:
+    from airflow.utils import timezone  # type: ignore[attr-defined]
+except ImportError:
+    from airflow.sdk import timezone
```

Comment:
```suggestion

try:
    from airflow.sdk import timezone
except ImportError:
    from airflow.utils import timezone  # type: ignore[attr-defined,no-redef]
```

The fallback is on the opposite side.

Reply:
Done !!